src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalVariantInputArgumentCollection.java

```diff
@@ -15,7 +15,7 @@
  */
 public class OptionalVariantInputArgumentCollection implements ArgumentCollectionDefinition {
 
-    @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME, shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME, doc = "One or more files containing variants", optional = true)
+    @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME, shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME, doc = "Variants file", optional = true)
```

Comment:
in another change in this review (OptionalFeatureInputArgumentCollection) the doc string started with lowercase. Here it starts with uppercase